tests/test_build/source/code.js

```diff
@@ -141,3 +141,11 @@ class ExampleClass {}
  * console.log(ExampleAttribute);
  */
 const ExampleAttribute = null;
+
+/**
+ * @param {Number} p1
+ * @param {Object} p2
+ * @param {String} p2.foo
```

Comment:
This deserves an example in the readme. Is this the usual approach to documenting destructured formal params in JSDoc?

Reply:
I added examples, but feel free to fix my english :)

Reply:
Better than my French! :-)